scripts/.check-license.ignore

```diff
@@ -5,6 +5,7 @@
 .*\.md$
 .*\.patch$
 .*\.pdf$
+.*\.yaml$
```

Comment:
YAML does support comments just like bash does, so I'd recommend reverting this line and adding the license to the top of the yaml files